app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1><%= @recipe.name %></h1>
+<p>Complexity: <%= @recipe.complexity %></p>
+<p>Genre: <%= @recipe.genre %></p>
+
+<h2>Ingredients:</h2>
+<ul>
+  <% @recipe.ingredients.each do |ingredient| %>
+    <li><%= ingredient.name %></li>
+  <% end %>
+</ul>
+
+<h3>Total Cost: <%= "$#{@total_cost}" %></p>
```

Comment:
Here is where we would call our `total_cost` method (see my comment in your recipe controller). 
```ruby
Total Cost: $<%= @recipe.total_cost %>
```